lib/elasticity/search.rb

```diff
@@ -28,11 +29,13 @@ def to_count_args
       end
 
       def to_search_args
-        { index: @index_name, type: @document_types, body: @body }
+        { index: @index_name, type: @document_types, body: @body }.reverse_merge(@search_args)
```

Comment:
very `nitpicky` because I  know we have activeSupport as a dependency, but the we could get the same results without needing reverse_merge,  I think, with
```
@search_args.merge({ index: @index_name, type: @document_types, body: @body })
```
and lessen our dependency on ActiveSupport
(similar comment for line 36)